### DISCUSSION TITLE
Uninstall quietly

### DISCUSSION THREAD

**Discussion by mephistophen** [valinet/ExplorerPatcher#2092] [Ideas]
### Discussed in https://github.com/valinet/ExplorerPatcher/discussions/2007

<div type='discussions-op-text'>

<sup>Originally posted by **mephistophen** September  6, 2023</sup>
Hello, do you know how to launch de command of uninstall ep_setup.exe /uninstall but quietly ?

I have to launch multiple uninstall of dozen of workstations because of the issue with Windows Update.

Thank you</div>

**Comment by mephistophen** [comment]
This is an Issue because, if you had 200 workstations with your product that crash explorer.exe in loop and you can't uninstall quietly with remote command (with Ivanti, scripting or SCCM for example) it's serious  !

**Reply by Poopooracoocoo** [reply]
It's still a feature request. The crashing, however, is an issue.
